### PR TITLE
Chrome 83+ supports CSS selector: :where() behind a flag

### PR DESCRIPTION
**Pull request by Schweinepriester**
Fixes Fyrd/caniuse#5481.

I've tested using <https://jsfiddle.net/simevidas/d1tLszx9/> from <https://bugs.chromium.org/p/chromium/issues/detail?id=808904#c7>:

image

Chromestatus: https://www.chromestatus.com/feature/4600991135563776

### CODE REVIEW

File: css/selectors/where.json [hunk 1]
```diff
@@ -7,7 +7,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:where",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "83",
```
Comment (ddbeck) [inline css/selectors/where.json:10]: This feature seems to be somewhat older than your test shows. It looks like [this change](https://storage.googleapis.com/chromium-find-releases-static/e4a.html#e4a8dda8926d36a02e3f3bc3d20d839bb71ede5c) renamed it from `is()` to `:where()` in Chrome 72.

```suggestion
              "version_added": "72",
```

File: css/selectors/where.json [hunk 2]
```diff
@@ -7,7 +7,14 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Experimental Web Platform Features",
```
Comment (ddbeck) [inline css/selectors/where.json:14]: For consistency, I'd suggest changing this. This hasn't been formalized yet, but see #6009 for discussion.

```suggestion
                  "name": "#enable-experimental-web-platform-features",
```

File: css/selectors/where.json [hunk 3]
```diff
@@ -7,7 +7,14 @@
...
+                  "value_to_set": "enabled"
```
Comment (ddbeck) [inline css/selectors/where.json:15]:
```suggestion
                  "value_to_set": "Enabled"
```

File: css/selectors/where.json [hunk 4]
```diff
@@ -7,7 +7,14 @@
...
+                }
+              ]
             },
             "chrome_android": {
               "version_added": false
```
Comment (ddbeck) [inline css/selectors/where.json:20]: Can you duplicate the data point above to Chrome for Android? They're nearly always the same

File: css/selectors/where.json (hunk 1 above)
Comment (Schweinepriester) [reply css/selectors/where.json]: You found the change, nice! Tried to but didnt find it...

**Comment by Schweinepriester** [comment]
Just to double check regarding the inconsistencies, I copied those from `:is()`: https://github.com/mdn/browser-compat-data/blob/master/css/selectors/is.json#L9

Given the tendency of #6009 I should change them? And maybe on top in another PR correct those in `:is()`?

File: css/selectors/where.json (hunk 4 above)
Comment (jpmedley) [reply css/selectors/where.json:20]: If they're not, that will be reflected in the Chrome Status entry.

**Comment by ddbeck** [comment]
I thought that, while we're editing the data anyway, it'd be nice to get some more consistency (I had done a little bit of work to fix #6009, but there's no PR yet). You don't have to change the  `flags` data (for this PR or for `:is()`), if you don't want to, but I would like to see the data copied to `chrome_android` before merging. Thank you!

**Comment by Schweinepriester** [comment]
Done I think :)

**Pull request merged**